lib/src/platform_client.dart

```diff
@@ -216,7 +217,7 @@ class PlatformClient {
   ///
   /// [cannotTalk] will let the agent know if the Explorer cannot talk at connection time.
   Future<Room> createServiceRequest(RoomHandler roomHandler,
-      {Position? position, String? message, Map<String, List<int>>? fileMap, bool? cannotTalk}) async {
+      {Position? position, String? message, Map<String, List<int>>? fileMap, bool? cannotTalk, int? accountId}) async {
```

Comment:
Add optional account ID so billing can be charged based on account ID with profile